packages/gatsby/src/utils/detect-hot-loader-to-use.ts

```diff
@@ -0,0 +1,35 @@
+import semver from "semver"
+
+// Fast refresh is supported as of React 16.8.
+// This package will do some sniffing to see if the current version of
+// React installed is greater than 16.8.
+export function detectHotLoaderToUse(): void {
+  // If the user has defined this, we don't want to do any package sniffing
+  if (process.env.GATSBY_HOT_LOADER) return
+
+  // Do some package sniffing to see if we can use fast-refresh if the user didn't
+  // specify a specific hot loader with the environment variable.
+  try {
+    const reactVersion = require(`react/package.json`).version
+    const parsed = semver.parse(reactVersion)
+
+    console.log(parsed)
```

Comment:
Not a lot of context on this but it looks good to me. However, should this and the console.debug below have been removed?